src/index.css

```diff
@@ -1,13 +1,201 @@
 :root {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Roboto Mono", monospace;
+  font-optical-sizing: auto;
+  font-weight: 400;
+  font-style: normal;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  margin: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  --light-pink: rgba(255, 192, 203, 0.775);
+  --medium-pink: rgb(255, 150, 175);
+  --dark-pink: rgb(250, 109, 142);
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.wrapper {
+  width: 100%;
+  margin: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.app-field {
+  max-width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.post-form {
+  background-color: lightgrey;
+  border: 2px solid black;
+  filter: drop-shadow(10px 15px);
+  margin: 15px;
+  width: 65%;
+  display: flex;
+  flex-direction: column;
+}
+
+.input-field {
+  margin: 20px 20px 0 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.text-field {
+  height: 60px;
+  width: 100%;
+  padding: 10px 5px;
+  margin: 5px 0;
+  resize: none;
+  box-sizing: border-box;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.characters-count {
+  font-size: 7px;
+  text-align: end;
+  margin: 0 20px 10px;
 }
+
+.red-text {
+  color: red;
+}
+
+.post-button {
+  background-color: var(--light-pink);
+  font-family: "Roboto Mono", monospace;
+  border: none;
+  border-radius: 20px;
+  margin: 0 20px 20px;
+  display: flex;
+  align-items: center;
+}
+
+.medium-pink-button {
+  background-color: var(--medium-pink);
+}
+
+.post-button:hover {
+  background-color: var(--dark-pink);
+  padding: 3px 7px;
+  margin-bottom: 16px;
+}
+
+.post-button:hover .heart{
+  opacity: 1;
+}
+
+.post-heart {
+  margin: 10px;
+}
+
+.thoughts-section {
+  width: 65%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.thought-box {
```

Comment:
Sometimes the characters go outside the thought-box, suggestion to maybe add word-wrap: break-word.

Reply:
Thanks, I will try that! :)